pkgs/by-name/op/openinfraquote-bin/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  autoPatchelfHook,
+  lib,
+  stdenv,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "openinfraquote-bin";
```

Comment:
bin is only added to the pname if there are two packages for the same software. We can drop it here but regardless we should set meta.sourceProvenance.